tsconfig.json

```diff
@@ -17,5 +17,5 @@
         "noEmit": true,
         "jsx": "react-jsx"
     },
-    "include": ["src"]
+    "include": ["src","decs.d.ts"]
```

Comment:
decs.d.ts is only used by files that are in src so you can put it in src. If you see that it could be used by another module, you could put it in commons-ui

Reply:
decs.d.ts file is already created in another PR(outside the src) so to avoid duplication I kept the it outside src.
We could  remove it inside src after.

Reply:
Main already contains decs.d.ts outside src.